src/clocksi_vnode.erl

```diff
@@ -91,30 +90,6 @@ read_data_item(Node, TxId, Key, Type, Updates) ->
             {error, Reason}
     end.
 
-
-%% @doc Return active transactions in prepare state with their preparetime for a given key
-%% should be run from same physical node
-get_active_txns_key(Key, Partition, TableName) ->
-    case ets:info(TableName) of
-        undefined ->
```

Comment:
I would recommend keeping keeping the check for ets:info, the tests pass because at startup there is a wait loop that checks the vnodes have started and loaded the ets tables.  If the wait loop isn't run then there will be crashes in other places, for example in the benchmarks.  You can directly move this code into the readitem fsm if you want.